openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -18,6 +18,7 @@
 from django.db.models.signals import post_save, post_delete
 from django.db.utils import IntegrityError
 from django.template import defaultfilters
+from jsonfield import JSONField
 
 from django.utils.functional import cached_property
 from model_utils.models import TimeStampedModel
```

Comment:
I think you missed bumping this model's version number:

https://github.com/edx/edx-platform/blob/d66d544ca255c7b174126e128e1999430b005640/openedx/core/djangoapps/content/course_overviews/models.py#L67-L68

Reply:
Yep, missed that. Fixed in fourth commit.